.github/workflows/message-discord.yaml

```diff
@@ -0,0 +1,29 @@
+name: Notify Discord on PR Events
+
+on:
+  pull_request:
+    types: [opened, closed]
+
+jobs:
+  notify-discord:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Send Discord notification
+        env:
+          DISCORD_WEBHOOK_URL: ${{ secrets.DISCORD_WEBHOOK_URL }}
+        run: |
+          # Construct the message based on the type of PR event
+          if [[ "${{ github.event.action }}" == "opened" ]]; then
+            MESSAGE="📢 A new pull request has been opened in \`${{ github.repository }}\` by \`${{ github.actor }}\`.\n🔗 [PR #${{ github.event.number }} - ${{ github.event.pull_request.title }}](${{ github.event.pull_request.html_url }})";
```

Comment:
nit: line breaks to make it more readable, but this isn't a big deal

e.g.

```bash
MESSAGE="📢 A new pull request has been opened in \`${{ github.repository }}\`"
MESSAGE+="by \`${{ github.actor }}\`.\n"
MESSAGE+="🔗 [PR #${{ github.event.number }} - ${{ github.event.pull_request.title }}](${{ github.event.pull_request.html_url }})";"
```

Reply:
Also though, you can do single quotes and just rely on the actual line break instead of using a `\n`

Reply:
10-4